src/esbmc/bmc.cpp

```diff
@@ -42,6 +40,10 @@
 #include <atomic>
 #include <goto-symex/witnesses.h>
 
+std::unordered_set<std::string> goto_functionst::reached_claims;
```

Comment:
Do we really need these global variables?

Reply:
It is the static member, must be defined and initialized once outside the class definition